similarity-data-impl/src/main/java/org/phenotips/data/similarity/script/PatientMatchingScriptService.java

```diff
@@ -55,7 +59,7 @@
     public void clearCache()
     {
         ((DefaultPatientSimilarityViewFactory) patientViewFactory).clearCache();
-        PatientGenotype.clearCache();
+        ((ExomiserExomeManager) exomeManager).clearCache();
```

Comment:
That's not nice, don't make assumptions about the implementation. Maybe add the `clearCache` method in `ExomeManager`?